INFO 2021-06-09 11:34:31,055 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-09 11:34:39,014 basehttp 154 "GET / HTTP/1.1" 200 4
INFO 2021-06-09 11:38:14,075 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-09 11:38:20,651 urls 27 info
INFO 2021-06-09 11:38:20,658 basehttp 154 "GET / HTTP/1.1" 200 4
INFO 2021-06-09 11:48:55,796 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-09 11:50:01,559 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-09 11:50:39,528 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-09 11:51:20,109 basehttp 154 "GET /static/index.html HTTP/1.1" 200 12002
INFO 2021-06-09 11:51:20,545 basehttp 154 "GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 140942
INFO 2021-06-09 11:51:20,606 basehttp 154 "GET /static/md_css/monokai.css HTTP/1.1" 200 4118
INFO 2021-06-09 11:51:20,633 basehttp 154 "GET /static/common/jquery.pagination.css HTTP/1.1" 200 523
INFO 2021-06-09 11:51:20,634 basehttp 154 "GET /static/common/common.css HTTP/1.1" 200 381
INFO 2021-06-09 11:51:20,646 basehttp 154 "GET /static/js/index.js HTTP/1.1" 200 530
INFO 2021-06-09 11:51:20,648 basehttp 154 "GET /static/js/common.js HTTP/1.1" 200 891
INFO 2021-06-09 11:51:20,650 basehttp 154 "GET /static/js/host.js HTTP/1.1" 200 35
INFO 2021-06-09 11:51:20,657 basehttp 154 "GET /static/js/jquery.pagination.min.js HTTP/1.1" 200 2935
INFO 2021-06-09 11:51:20,661 basehttp 154 "GET /static/js/axios-0.18.0.min.js HTTP/1.1" 200 12941
INFO 2021-06-09 11:51:20,696 basehttp 154 "GET /static/js/vue-2.5.16.js HTTP/1.1" 200 289307
INFO 2021-06-09 11:51:20,709 basehttp 154 "GET /static/js/jquery-1.12.4.min.js HTTP/1.1" 200 97163
INFO 2021-06-09 11:51:25,992 basehttp 154 "GET /static/img/1.jpg HTTP/1.1" 200 78682
INFO 2021-06-09 11:52:02,407 urls 27 info
INFO 2021-06-09 11:52:02,427 basehttp 154 "GET / HTTP/1.1" 200 4
INFO 2021-06-09 12:27:23,021 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-09 12:30:41,893 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-09 12:31:41,362 autoreload 597 Watching for file changes with StatReloader
WARNING 2021-06-09 12:32:54,649 log 222 Not Found: /
WARNING 2021-06-09 12:32:54,695 basehttp 154 "GET / HTTP/1.1" 404 2092
WARNING 2021-06-09 12:32:55,232 log 222 Not Found: /favicon.ico
WARNING 2021-06-09 12:32:55,232 basehttp 154 "GET /favicon.ico HTTP/1.1" 404 2143
INFO 2021-06-09 12:34:01,697 basehttp 154 "GET /register HTTP/1.1" 301 0
INFO 2021-06-09 12:34:02,592 basehttp 154 "GET /register/ HTTP/1.1" 200 5217
INFO 2021-06-09 12:34:02,898 basehttp 154 "GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 140942
INFO 2021-06-09 12:34:02,927 basehttp 154 "GET /static/js/host.js HTTP/1.1" 200 35
INFO 2021-06-09 12:34:02,937 basehttp 154 "GET /static/img/image_code.png HTTP/1.1" 200 3483
INFO 2021-06-09 12:34:02,937 basehttp 154 "GET /static/js/common.js HTTP/1.1" 200 891
INFO 2021-06-09 12:34:02,958 basehttp 154 "GET /static/js/axios-0.18.0.min.js HTTP/1.1" 200 12941
INFO 2021-06-09 12:34:02,965 basehttp 154 "GET /static/js/register.js HTTP/1.1" 200 6407
INFO 2021-06-09 12:46:45,963 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-09 12:47:12,100 basehttp 154 "GET /register/ HTTP/1.1" 200 5208
INFO 2021-06-09 12:47:12,173 basehttp 154 "GET /static/js/axios-0.18.0.min.js HTTP/1.1" 304 0
INFO 2021-06-09 12:47:12,173 basehttp 154 "GET /static/js/vue-2.5.16.js HTTP/1.1" 304 0
INFO 2021-06-09 12:47:12,176 basehttp 154 "GET /static/js/host.js HTTP/1.1" 304 0
INFO 2021-06-09 12:47:12,177 basehttp 154 "GET /static/js/common.js HTTP/1.1" 304 0
INFO 2021-06-09 12:47:12,181 basehttp 154 "GET /static/js/register.js HTTP/1.1" 304 0
INFO 2021-06-09 12:47:12,488 basehttp 154 "GET /static/img/image_code.png HTTP/1.1" 304 0
<<<<<<< HEAD
INFO 2021-06-09 14:29:23,698 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-09 14:32:03,348 autoreload 217 /home/kura/PycharmProjects/my_python_project/django/blog/blog/users/views.py changed, reloading.
INFO 2021-06-09 14:32:05,044 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-09 14:32:10,712 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-09 14:35:42,740 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-09 14:40:52,793 autoreload 597 Watching for file changes with StatReloader
ERROR 2021-06-09 14:43:38,489 log 222 Internal Server Error: /imagecode/
Traceback (most recent call last):
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 97, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/kura/PycharmProjects/my_python_project/django/blog/blog/users/views.py", line 42, in get
    redis_conn = get_redis_connection('defalut')
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django_redis/__init__.py", line 12, in get_redis_connection
    cache = caches[alias]
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/cache/__init__.py", line 75, in __getitem__
    raise InvalidCacheBackendError(
django.core.cache.backends.base.InvalidCacheBackendError: Could not find config for 'defalut' in settings.CACHES
ERROR 2021-06-09 14:44:01,835 basehttp 154 "GET /imagecode/?uuid=123 HTTP/1.1" 500 96030
INFO 2021-06-09 14:44:51,338 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-09 14:45:18,905 autoreload 597 Watching for file changes with StatReloader
ERROR 2021-06-09 14:48:08,844 log 222 Internal Server Error: /imagecode/
Traceback (most recent call last):
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 97, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/kura/PycharmProjects/my_python_project/django/blog/blog/users/views.py", line 42, in get
    redis_conn = get_redis_connection('defalut')
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django_redis/__init__.py", line 12, in get_redis_connection
    cache = caches[alias]
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/cache/__init__.py", line 75, in __getitem__
    raise InvalidCacheBackendError(
django.core.cache.backends.base.InvalidCacheBackendError: Could not find config for 'defalut' in settings.CACHES
ERROR 2021-06-09 14:49:23,516 basehttp 154 "GET /imagecode/?uuid=123 HTTP/1.1" 500 96152
ERROR 2021-06-09 14:50:50,119 log 222 Internal Server Error: /imagecode/
Traceback (most recent call last):
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 97, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/kura/PycharmProjects/my_python_project/django/blog/blog/users/views.py", line 42, in get
    redis_conn = get_redis_connection('defalut')
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django_redis/__init__.py", line 12, in get_redis_connection
    cache = caches[alias]
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/cache/__init__.py", line 75, in __getitem__
    raise InvalidCacheBackendError(
django.core.cache.backends.base.InvalidCacheBackendError: Could not find config for 'defalut' in settings.CACHES
ERROR 2021-06-09 14:51:16,860 basehttp 154 "GET /imagecode/?uuid=123 HTTP/1.1" 500 96142
ERROR 2021-06-09 14:52:06,376 log 222 Internal Server Error: /imagecode/
Traceback (most recent call last):
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 97, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/kura/PycharmProjects/my_python_project/django/blog/blog/users/views.py", line 42, in get
    redis_conn = get_redis_connection('defalut')
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django_redis/__init__.py", line 12, in get_redis_connection
    cache = caches[alias]
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/cache/__init__.py", line 75, in __getitem__
    raise InvalidCacheBackendError(
django.core.cache.backends.base.InvalidCacheBackendError: Could not find config for 'defalut' in settings.CACHES
ERROR 2021-06-09 14:52:06,384 basehttp 154 "GET /imagecode/?uuid=123 HTTP/1.1" 500 96169
INFO 2021-06-09 14:52:13,806 autoreload 597 Watching for file changes with StatReloader
ERROR 2021-06-09 14:54:06,573 log 222 Internal Server Error: /imagecode/
Traceback (most recent call last):
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 97, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/kura/PycharmProjects/my_python_project/django/blog/blog/users/views.py", line 42, in get
    redis_conn = get_redis_connection('defalut')
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django_redis/__init__.py", line 12, in get_redis_connection
    cache = caches[alias]
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/cache/__init__.py", line 75, in __getitem__
    raise InvalidCacheBackendError(
django.core.cache.backends.base.InvalidCacheBackendError: Could not find config for 'defalut' in settings.CACHES
ERROR 2021-06-09 14:54:33,702 basehttp 154 "GET /imagecode/?uuid=123 HTTP/1.1" 500 96143
ERROR 2021-06-09 14:56:21,275 log 222 Internal Server Error: /imagecode/
Traceback (most recent call last):
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 97, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/kura/PycharmProjects/my_python_project/django/blog/blog/users/views.py", line 42, in get
    redis_conn = get_redis_connection('defalut')
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django_redis/__init__.py", line 12, in get_redis_connection
    cache = caches[alias]
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/cache/__init__.py", line 75, in __getitem__
    raise InvalidCacheBackendError(
django.core.cache.backends.base.InvalidCacheBackendError: Could not find config for 'defalut' in settings.CACHES
ERROR 2021-06-09 14:56:21,287 basehttp 154 "GET /imagecode/?uuid=123 HTTP/1.1" 500 96196
ERROR 2021-06-09 14:59:00,240 log 222 Internal Server Error: /imagecode/
Traceback (most recent call last):
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 97, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/kura/PycharmProjects/my_python_project/django/blog/blog/users/views.py", line 42, in get
    redis_conn = get_redis_connection('defalut')
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django_redis/__init__.py", line 12, in get_redis_connection
    cache = caches[alias]
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/cache/__init__.py", line 75, in __getitem__
    raise InvalidCacheBackendError(
django.core.cache.backends.base.InvalidCacheBackendError: Could not find config for 'defalut' in settings.CACHES
ERROR 2021-06-09 14:59:05,627 basehttp 154 "GET /imagecode/?uuid=123 HTTP/1.1" 500 96176
ERROR 2021-06-09 14:59:26,661 log 222 Internal Server Error: /imagecode/
Traceback (most recent call last):
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 97, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/kura/PycharmProjects/my_python_project/django/blog/blog/users/views.py", line 42, in get
    redis_conn = get_redis_connection('defalut')
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django_redis/__init__.py", line 12, in get_redis_connection
    cache = caches[alias]
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/cache/__init__.py", line 75, in __getitem__
    raise InvalidCacheBackendError(
django.core.cache.backends.base.InvalidCacheBackendError: Could not find config for 'defalut' in settings.CACHES
ERROR 2021-06-09 14:59:26,662 basehttp 154 "GET /imagecode/?uuid=123 HTTP/1.1" 500 96154
ERROR 2021-06-09 14:59:30,175 log 222 Internal Server Error: /imagecode/
Traceback (most recent call last):
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 97, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/kura/PycharmProjects/my_python_project/django/blog/blog/users/views.py", line 42, in get
    redis_conn = get_redis_connection('defalut')
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django_redis/__init__.py", line 12, in get_redis_connection
    cache = caches[alias]
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/cache/__init__.py", line 75, in __getitem__
    raise InvalidCacheBackendError(
django.core.cache.backends.base.InvalidCacheBackendError: Could not find config for 'defalut' in settings.CACHES
ERROR 2021-06-09 14:59:30,176 basehttp 154 "GET /imagecode/?uuid=123 HTTP/1.1" 500 96176
INFO 2021-06-09 15:04:44,246 autoreload 597 Watching for file changes with StatReloader
ERROR 2021-06-09 15:05:05,833 log 222 Internal Server Error: /imagecode/
Traceback (most recent call last):
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/views/generic/base.py", line 97, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/kura/PycharmProjects/my_python_project/django/blog/blog/users/views.py", line 42, in get
    redis_conn = get_redis_connection('defalut')
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django_redis/__init__.py", line 12, in get_redis_connection
    cache = caches[alias]
  File "/home/kura/.virtualenvs/blog/lib/python3.8/site-packages/django/core/cache/__init__.py", line 75, in __getitem__
    raise InvalidCacheBackendError(
django.core.cache.backends.base.InvalidCacheBackendError: Could not find config for 'defalut' in settings.CACHES
ERROR 2021-06-09 15:05:05,834 basehttp 154 "GET /imagecode/?uuid=123 HTTP/1.1" 500 91668
INFO 2021-06-09 15:06:28,304 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-09 15:06:35,736 basehttp 154 "GET /imagecode/?uuid=123 HTTP/1.1" 200 3304
INFO 2021-06-09 15:06:47,760 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-09 15:06:53,364 basehttp 154 "GET /imagecode/?uuid=123 HTTP/1.1" 200 2917
INFO 2021-06-09 15:07:41,063 basehttp 154 "GET /imagecode/?uuid=123 HTTP/1.1" 200 3210
INFO 2021-06-09 15:12:20,526 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-09 15:12:24,219 basehttp 154 "GET /imagecode/?uuid=123 HTTP/1.1" 200 2761
INFO 2021-06-09 15:12:26,770 basehttp 154 "GET /imagecode/?uuid=123 HTTP/1.1" 200 3395
INFO 2021-06-09 15:12:41,825 basehttp 154 "GET /register/ HTTP/1.1" 200 5197
INFO 2021-06-09 15:12:44,286 basehttp 154 "GET /imagecode/?uuid=094c0dcd-5937-42b8-ba52-e3c7e70d8f16 HTTP/1.1" 200 3275
INFO 2021-06-09 15:12:52,410 basehttp 154 "GET /imagecode/?uuid=964e917d-b9ca-4803-8db7-b1508c08c796 HTTP/1.1" 200 2850
INFO 2021-06-09 15:12:53,400 basehttp 154 "GET /imagecode/?uuid=c1b7c1be-3a7f-45bc-8bd3-ab4a50b91c03 HTTP/1.1" 200 3405
INFO 2021-06-09 15:12:54,108 basehttp 154 "GET /imagecode/?uuid=c9b1efdb-a6e6-42d9-aaa7-06cef7827c25 HTTP/1.1" 200 3790
INFO 2021-06-09 15:12:54,614 basehttp 154 "GET /imagecode/?uuid=e1d7086b-3e62-465b-a37b-e6eba12f98f6 HTTP/1.1" 200 3108
INFO 2021-06-10 10:26:20,151 autoreload 597 Watching for file changes with StatReloader
WARNING 2021-06-10 10:26:36,768 log 222 Not Found: /
WARNING 2021-06-10 10:26:36,768 basehttp 154 "GET / HTTP/1.1" 404 2267
INFO 2021-06-10 10:26:54,542 basehttp 154 "GET /register/ HTTP/1.1" 200 5197
INFO 2021-06-10 10:26:56,123 basehttp 154 "GET /imagecode/?uuid=fd4f1e42-9329-48dc-82b6-03b0b29d3aac HTTP/1.1" 200 3648
INFO 2021-06-10 10:27:28,325 basehttp 154 "GET /imagecode/?uuid=7402e337-adb5-455f-9ab7-1b4ace86d3d5 HTTP/1.1" 200 3068
INFO 2021-06-10 10:31:22,153 autoreload 217 /home/kura/PycharmProjects/my_python_project/django/blog/blog/users/urls.py changed, reloading.
INFO 2021-06-10 10:31:24,192 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-10 10:32:35,818 autoreload 217 /home/kura/PycharmProjects/my_python_project/django/blog/blog/users/urls.py changed, reloading.
INFO 2021-06-10 10:32:36,773 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-10 10:32:48,490 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-10 10:33:31,405 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-10 10:33:40,757 basehttp 154 "GET /register/ HTTP/1.1" 200 5197
INFO 2021-06-10 10:33:40,786 basehttp 154 "GET /static/js/host.js HTTP/1.1" 304 0
INFO 2021-06-10 10:33:40,786 basehttp 154 "GET /static/js/vue-2.5.16.js HTTP/1.1" 304 0
INFO 2021-06-10 10:33:40,792 basehttp 154 "GET /static/js/common.js HTTP/1.1" 304 0
INFO 2021-06-10 10:33:40,797 basehttp 154 "GET /static/js/register.js HTTP/1.1" 304 0
INFO 2021-06-10 10:33:40,828 basehttp 154 "GET /static/js/axios-0.18.0.min.js HTTP/1.1" 304 0
INFO 2021-06-10 10:33:40,904 basehttp 154 "GET /imagecode/?uuid=ff983399-08f1-4418-bfe2-e12fb1a1dc51 HTTP/1.1" 200 3533
INFO 2021-06-10 10:34:02,730 basehttp 154 "GET /imagecode/?uuid=73e6a5f0-3f0f-4537-a8ce-62d3adca1b65 HTTP/1.1" 200 2878
INFO 2021-06-10 10:36:53,841 views 104 368166
INFO 2021-06-10 10:37:17,587 basehttp 154 "GET /smscode/?mobile=15708170445&image_code=ZKIS&uuid=73e6a5f0-3f0f-4537-a8ce-62d3adca1b65 HTTP/1.1" 200 69
INFO 2021-06-10 12:08:03,007 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-10 12:09:22,564 basehttp 154 "GET /imagecode/?uuid=b2d3e8c5-13fc-4e00-a755-59f12bddac10 HTTP/1.1" 200 3342
INFO 2021-06-10 12:09:28,927 views 159 237384
INFO 2021-06-10 12:09:29,224 basehttp 154 "GET /smscode/?mobile=15708170445&image_code=6asr&uuid=b2d3e8c5-13fc-4e00-a755-59f12bddac10 HTTP/1.1" 200 69
WARNING 2021-06-10 12:10:09,118 log 222 Forbidden (CSRF cookie not set.): /register/
WARNING 2021-06-10 12:10:09,119 basehttp 154 "POST /register/ HTTP/1.1" 403 2868
INFO 2021-06-10 12:13:10,489 autoreload 597 Watching for file changes with StatReloader
WARNING 2021-06-10 12:13:27,631 log 222 Forbidden (CSRF cookie not set.): /register/
WARNING 2021-06-10 12:13:27,631 basehttp 154 "POST /register/ HTTP/1.1" 403 2868
WARNING 2021-06-10 12:13:30,707 log 222 Forbidden (CSRF cookie not set.): /register/
WARNING 2021-06-10 12:13:30,708 basehttp 154 "POST /register/ HTTP/1.1" 403 2868
INFO 2021-06-10 12:13:36,382 basehttp 154 "GET /register/ HTTP/1.1" 200 5339
INFO 2021-06-10 12:13:36,793 basehttp 154 "GET /imagecode/?uuid=cbfbf70d-7d94-404d-90ef-b342d31d7aa7 HTTP/1.1" 200 2983
INFO 2021-06-10 12:13:52,806 basehttp 154 "GET /imagecode/?uuid=98f6ec00-c239-479e-add7-927ef13bf0a9 HTTP/1.1" 200 3415
INFO 2021-06-10 12:14:03,009 views 159 961311
INFO 2021-06-10 12:14:03,277 basehttp 154 "GET /smscode/?mobile=15708170445&image_code=S6LO&uuid=98f6ec00-c239-479e-add7-927ef13bf0a9 HTTP/1.1" 200 69
WARNING 2021-06-10 12:14:15,965 log 222 Bad Request: /register/
WARNING 2021-06-10 12:14:15,966 basehttp 154 "POST /register/ HTTP/1.1" 400 31
=======
>>>>>>> a1d157b356422f50048ceddc3e2a422e0852a3cb
INFO 2021-06-10 12:43:46,396 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-10 12:45:08,751 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-10 12:47:00,021 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-10 12:50:56,420 autoreload 597 Watching for file changes with StatReloader
WARNING 2021-06-10 12:51:03,728 log 222 Bad Request: /register/
WARNING 2021-06-10 12:51:03,729 basehttp 154 "POST /register/ HTTP/1.1" 400 24
INFO 2021-06-10 12:51:09,166 basehttp 154 "GET /register/ HTTP/1.1" 200 5617
INFO 2021-06-10 12:51:09,413 basehttp 154 "GET /static/img/image_code.png HTTP/1.1" 200 3483
WARNING 2021-06-10 12:51:09,711 log 222 Not Found: /imagecode/
WARNING 2021-06-10 12:51:09,711 basehttp 154 "GET /imagecode/?uuid=a896dc53-7ad8-4142-ac71-e02591313624 HTTP/1.1" 404 2182
INFO 2021-06-10 12:52:34,003 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-10 12:52:38,641 basehttp 154 "GET /register/ HTTP/1.1" 200 5477
INFO 2021-06-10 12:52:38,673 basehttp 154 "GET /static/js/axios-0.18.0.min.js HTTP/1.1" 304 0
INFO 2021-06-10 12:52:38,676 basehttp 154 "GET /static/js/host.js HTTP/1.1" 304 0
INFO 2021-06-10 12:52:38,681 basehttp 154 "GET /static/js/register.js HTTP/1.1" 304 0
INFO 2021-06-10 12:52:38,682 basehttp 154 "GET /static/js/vue-2.5.16.js HTTP/1.1" 304 0
INFO 2021-06-10 12:52:38,682 basehttp 154 "GET /static/js/common.js HTTP/1.1" 304 0
INFO 2021-06-10 12:52:38,697 basehttp 154 "GET /static/img/image_code.png HTTP/1.1" 304 0
WARNING 2021-06-10 12:52:38,723 log 222 Not Found: /imagecode/
WARNING 2021-06-10 12:52:38,723 basehttp 154 "GET /imagecode/?uuid=799eb779-3af1-44f3-a881-d2a5f04a2047 HTTP/1.1" 404 2182
INFO 2021-06-10 12:52:41,498 basehttp 154 "GET /register/ HTTP/1.1" 200 5477
WARNING 2021-06-10 12:52:41,591 log 222 Not Found: /imagecode/
WARNING 2021-06-10 12:52:41,591 basehttp 154 "GET /imagecode/?uuid=30348569-40a9-4272-951f-16b9588fcd7e HTTP/1.1" 404 2182
WARNING 2021-06-10 12:52:48,180 log 222 Not Found: /imagecode/
WARNING 2021-06-10 12:52:48,180 basehttp 154 "GET /imagecode/?uuid=a0e0be20-778e-442a-970f-3aeb32ccd077 HTTP/1.1" 404 2182
WARNING 2021-06-10 12:52:48,870 log 222 Not Found: /imagecode/
WARNING 2021-06-10 12:52:48,871 basehttp 154 "GET /imagecode/?uuid=f4642770-d154-494c-bb51-bad024ab59fb HTTP/1.1" 404 2182
WARNING 2021-06-10 12:52:49,581 log 222 Not Found: /imagecode/
WARNING 2021-06-10 12:52:49,581 basehttp 154 "GET /imagecode/?uuid=2a2e996f-68c6-40f8-9cdf-dc515e6f591b HTTP/1.1" 404 2182
WARNING 2021-06-10 12:52:50,133 log 222 Not Found: /imagecode/
WARNING 2021-06-10 12:52:50,133 basehttp 154 "GET /imagecode/?uuid=a4ac765a-f2dc-4bbe-8742-c9f1ed1ba574 HTTP/1.1" 404 2182
INFO 2021-06-10 12:53:59,049 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-10 12:54:13,434 basehttp 154 "GET /register/ HTTP/1.1" 200 5476
INFO 2021-06-10 12:54:13,468 basehttp 154 "GET /static/js/vue-2.5.16.js HTTP/1.1" 304 0
INFO 2021-06-10 12:54:13,474 basehttp 154 "GET /static/js/register.js HTTP/1.1" 304 0
INFO 2021-06-10 12:54:13,478 basehttp 154 "GET /static/js/common.js HTTP/1.1" 304 0
INFO 2021-06-10 12:54:13,479 basehttp 154 "GET /static/js/host.js HTTP/1.1" 304 0
INFO 2021-06-10 12:54:13,482 basehttp 154 "GET /static/js/axios-0.18.0.min.js HTTP/1.1" 304 0
INFO 2021-06-10 12:54:13,499 basehttp 154 "GET /static/img/image_code.png HTTP/1.1" 304 0
WARNING 2021-06-10 12:54:13,543 log 222 Not Found: /imagecode/
WARNING 2021-06-10 12:54:13,544 basehttp 154 "GET /imagecode/?uuid=b6312a47-7c8e-4c65-8d46-64c2102f6579 HTTP/1.1" 404 2182
INFO 2021-06-10 12:54:19,551 basehttp 154 "GET /register/ HTTP/1.1" 200 5476
WARNING 2021-06-10 12:54:19,646 log 222 Not Found: /imagecode/
WARNING 2021-06-10 12:54:19,646 basehttp 154 "GET /imagecode/?uuid=f0ff01df-6ab3-4304-b2a6-468c12c3f530 HTTP/1.1" 404 2182
WARNING 2021-06-10 12:54:22,878 log 222 Not Found: /imagecode/
WARNING 2021-06-10 12:54:22,879 basehttp 154 "GET /imagecode/?uuid=9be7079b-e6c3-423d-9499-0173c5e9b341 HTTP/1.1" 404 2182
WARNING 2021-06-10 12:54:23,560 log 222 Not Found: /imagecode/
WARNING 2021-06-10 12:54:23,561 basehttp 154 "GET /imagecode/?uuid=d499903d-0b4b-4a08-af40-c6c09fa8063d HTTP/1.1" 404 2182
WARNING 2021-06-10 12:54:27,401 log 222 Not Found: /imagecode/
WARNING 2021-06-10 12:54:27,401 basehttp 154 "GET /imagecode/?uuid=76802c1c-611a-4025-a0a4-3bf370b7d3c1 HTTP/1.1" 404 2182
WARNING 2021-06-10 12:54:28,154 log 222 Not Found: /imagecode/
WARNING 2021-06-10 12:54:28,154 basehttp 154 "GET /imagecode/?uuid=31561793-5ae2-4607-92ad-79c157005d3e HTTP/1.1" 404 2182
WARNING 2021-06-10 12:54:31,519 log 222 Not Found: /imagecode/
WARNING 2021-06-10 12:54:31,520 basehttp 154 "GET /imagecode/?uuid=065ee442-a299-496d-8dc4-24d7095c74c6 HTTP/1.1" 404 2182
WARNING 2021-06-10 12:54:33,688 log 222 Not Found: /imagecode/
WARNING 2021-06-10 12:54:33,688 basehttp 154 "GET /imagecode/?uuid=92aeb80e-b338-44c4-b468-93f30ed7acf4 HTTP/1.1" 404 2182
INFO 2021-06-10 13:03:18,525 autoreload 597 Watching for file changes with StatReloader
INFO 2021-06-10 13:03:21,804 basehttp 154 "GET /register/ HTTP/1.1" 200 5379
INFO 2021-06-10 13:03:21,845 basehttp 154 "GET /static/js/vue-2.5.16.js HTTP/1.1" 304 0
INFO 2021-06-10 13:03:21,863 basehttp 154 "GET /static/js/common.js HTTP/1.1" 304 0
INFO 2021-06-10 13:03:21,868 basehttp 154 "GET /static/js/axios-0.18.0.min.js HTTP/1.1" 304 0
INFO 2021-06-10 13:03:21,869 basehttp 154 "GET /static/js/host.js HTTP/1.1" 304 0
INFO 2021-06-10 13:03:21,878 basehttp 154 "GET /static/js/register.js HTTP/1.1" 304 0
INFO 2021-06-10 13:03:21,882 basehttp 154 "GET /static/img/image_code.png HTTP/1.1" 304 0
